controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/resource/Tasks.java

```diff
@@ -76,9 +77,10 @@ public Response get(@PathParam("id") Long id) {
    */
   @GET
   @Produces(MediaType.APPLICATION_JSON)
-  public Response findTasks(@QueryParam("clusterName") Optional<String> clusterName,
-                              @QueryParam("state") Optional<TaskState> state) {
-    List<Task> result = taskQueue.peekTasks(clusterName.orElse(null),
+  public Response findTasks(@QueryParam("namespace") Optional<String> namespace,
+                            @QueryParam("clusterName") Optional<String> clusterName,
+                            @QueryParam("state") Optional<TaskState> state) {
+    List<Task> result = taskQueue.peekTasks(new Cluster(namespace.get(), clusterName.get()),
                                             state.map(TaskState::intValue).orElse(null));
```

Comment:
what if namespace and clusterName is not set in the query parameter?

```get(): 
If a value is present in this Optional, returns the value, otherwise throws NoSuchElementException.```